plugins/social_network/social_network.cpp

```diff
@@ -319,6 +326,7 @@ namespace golos {
                 pimpl->database().post_apply_operation.connect([&](const operation_notification &note) {
                     pimpl->on_operation(note);
                 });
+#ifndef IS_LOW_MEM
```

Comment:
Please, move this `ifndef`  4 lines up  - before line: `auto &db = pimpl->database();`
1. It will disable additional connection in boost::signal - small performance improvement.
2. It will remove compilation warning: `warning: unused variable ‘db’ [-Wunused-variable]`